### ISSUE TITLE
Multiple uploads

### ISSUE THREAD

**Issue by viv3kk** [andreipfeiffer/react-native-webview-android-file-upload#5]
This is pretty awesome module and after 3 weeks of research I have finalised this for my project. It would be great if we have multiple upload feature.

**Comment by andreipfeiffer** [comment]
The `multiple` attribute should be read from the `input` element and allow multiple selection from gallery.

@viv3kk you see this working only when you select existing files, right?

**Comment by andreipfeiffer** [comment]
@viv3kk I might have an update for you this weekend. I've managed to make multiple uploads work on my phone. I'll have to prepare a new release.

**Comment by viv3kk** [comment]
> I might have an update for you this weekend. I've managed to make multiple uploads work on my phone. I'll have to prepare a new release.
Hi, you share the tweaks so that i use it , in my current module

**Comment by viv3kk** [comment]
> The multiple attribute should be read from the input element and allow multiple selection from gallery.


I am using react-dropzone library which has multiple upload by default

**Comment by andreipfeiffer** [comment]
@viv3kk I will make a new release this weekend so you can use it directly. The code changes are significant and in many locations to paste them here.

**Comment by andreipfeiffer** [comment]
Added in [1.3.0](https://github.com/andreipfeiffer/react-native-webview-android-file-upload/releases/tag/1.3.0). @viv3kk please check it out.

**Comment by viv3kk** [comment]
> Added in 1.3.0. @viv3kk please check it out.

Will check this today and will be able to confirm it by tomorrow. Thanks as lot, it helping me big time.

**Comment by viv3kk** [comment]
Its not working for me.. This is what I did...
--> deleted node_modules/react-native-webview-android-file-upload folder
--> npm install latest version 1.3.0
Tried to upload a photo..but not able to select multiple photos..

**Comment by andreipfeiffer** [comment]
Does your `<input />` has the `multiple` attribute?
Also, please make sure that the latest version is in installed.
There are no other changes required, only that you re-compile your project.

If still not working, can you please tell me on what device are you testing?
I've only tested on my HTC m10 with Android 8, but will test with more devices today.

**Comment by viv3kk** [comment]
> Does your <input /> has the multiple attribute? - **yes**
Devices - Moto G Turbo (Android 7.1) and One plus One (Android 8)

Today I will build the code from start..

**Comment by andreipfeiffer** [comment]
Multiple file selection also works on:
* Samsung Galaxy S4 with Android 5.0.1
* OnePlus3T with Android 8.0

**Comment by andreipfeiffer** [comment]
@viv3kk did you manage to make it work?

**Comment by viv3kk** [comment]
For your reference:
https://youtu.be/EmBcPuvOVug

Kindly have a look... cannot select multiple files and is unable to select files from directory of file chooser

**Comment by viv3kk** [comment]
Hi, I tried to your example with multiple upload (which works) and then tried my actual url. But it didnt worked. I also have multiple property for my input.
My source url has http and urs is https ...does this makes a difference

**Comment by andreipfeiffer** [comment]
Hmmm, don't think that http has to do with this, but I'll have to double check.
Can you give me your url, if it's public?
Or the html content of the page if not?
It would be best to put your app in a repo, so I can try it out and debug it. Or send me a zip file with it.
I'll be on vacation till next week.

**Comment by viv3kk** [comment]
I create a scenario for this case and let you know..

**Comment by viv3kk** [comment]
So today I tried the build on samsung S5 ...and multiple uploads is working on this phone. Now this Sumsung phone is not a rooted phone while earlier I was using my own rooted devices. will this make a any difference.

**Comment by andreipfeiffer** [comment]
Hmmm, could be, I have no experience with rooted devices, so I don't know what are their limitations.
So, if this is working with a standard Android phone, could I close this issue?

**Comment by viv3kk** [comment]
Yes please do... I will testing with other phones for a while and let u know if there something which can be improved. Thanks a lot for this module.
